Components/Pages/Goals.js

```diff
@@ -31,6 +31,7 @@ const Goals = ({ navigation }) => {
   const { updateGoal } = useGoalUpdate();
   const { removeGoal } = useGoalDelete();
   refetch();
+setTimeout(refetch,2000)
```

Comment:
these two really should be replaced with useEffect:

- pass in refetch() for the callback (1st param)
- pass in [] to fetch once on render (2nd param) - same as componendDidMount()
- pass in [goals] to fetch only when goals variable changes (2nd param) - same as componendDidUpdate()